ntc_templates/templates/huawei_vrp_display_lldp_neighbor.textfsm

```diff
@@ -1,77 +1,103 @@
-Value Required LOCAL_INTERFACE (\S+)
-Value CHASSIS_ID (.+?)
+Value Required,Key LOCAL_INTERFACE (\S.+\S)
+Value CHASSIS_TYPE (\S.+\S)
+Value CHASSIS_ID (\S.+\S)
+Value PORT_ID_TYPE (\S.+\S)
 Value MANUFACTURER (.*?)
 Value PLATFORM (.*?)
 Value NEIGHBOR_PORT_ID (.*?)
-Value NEIGHBOR_INTERFACE (.*?)
-Value NEIGHBOR_NAME (.+?)
-Value List NEIGHBOR_DESCRIPTION (.*)
+Value NEIGHBOR_INTERFACE (\S.+\S)
+Value NEIGHBOR_NAME (\S.+\S)
+Value List NEIGHBOR_DESCRIPTION (\S.+\S)
+Value SYSTEM_CAPABILITIES_SUPPORTED (\S.+\S)
 Value CAPABILITIES (.*?)
+Value MGMT_ADDRESS_TYPE (\S+)
 Value MGMT_ADDRESS (\S+)
+Value EXPIRED_TIME (\d+)
 Value VLAN_ID (\d+)
+Value VLAN_1_NAME (\S.+\S)	
 Value SERIAL (\S+)
+Value OID (\S+)
+Value AUTONEGOTIATION_SUPPORTED (\S+)
+Value AUTONEGOTIATION_ENABLED (\S+)
+Value OPERATION_MODE (\S.+\S)
+Value POWER_PORT_CLASS (\S+)
+Value PSE_POWER_SUPPORTED (\S+)
+Value PSE_POWER_ENABLED (\S+)
+Value PSE_PAIRS_CONTROL_ABILITY (\S+)
+Value POWER_PAIRS (\S+)
+Value PORT_POWER_CLASSIFICATION (\S+)
+Value LINK_AGGREGATION_SUPPORTED (\S+)
+Value LINK_AGGREGATION_ENABLED (\S+)
+Value AGGREGATION_PORT_ID (\d+)
+Value MAXIMUM_FRAME_SIZE (\d+)
+Value MED_DEVICE_CLASS (\S.+\S)
+Value FIRMWARE_REVISION (\S.+\S)
+Value HARDWARE_REVISION (\S.+\S)
+
```

Comment:
I believe it's a single blank line after all of the Value definitions.
Minor, yes.

```suggestion
```